### PR TITLE
Add reporting extra examples contributing

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

We also want to know how our library is used. So I added the reporting example to contributing document.

 

### Details

- None

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ the community implement that functionality. For example, if you would like a
 reader for a specific file format, please provide a link to documentation of
 that file format and possibly provide some sample files with screenshots to work
 with. We will use the issue thread as a place to discuss and provide feedback.
+We also want to know how our library is used.
+If you use this library or find a great example of
+[using pyvista](https://github.com/pyvista/pyvista/network/dependents),
+you can also report it as *Feature Request* .
```
Comment (adeak) [inline CONTRIBUTING.md:60]:
```suggestion
If you use this library or find a great example
[using PyVista](https://github.com/pyvista/pyvista/network/dependents),
you can also report it as a *Feature Request*.
```

File: doc/extras/developer_notes.rst [hunk 2]
```diff
@@ -69,6 +69,10 @@ specific file format, please provide a link to documentation of that
 file format and possibly provide some sample files with screenshots to
 work with. We will use the issue thread as a place to discuss and
 provide feedback.
+We also want to know how our library is used.
+If you use this library or find a great example of `using
+pyvista <https://github.com/pyvista/pyvista/network/dependents>`__ ,
+you can also report it as *Feature Request* .
```
Comment (adeak) [inline doc/extras/developer_notes.rst:75]:
```suggestion
If you use this library or find a great example `using
PyVista <https://github.com/pyvista/pyvista/network/dependents>`__,
you can also report it as a *Feature Request*.
```

**Comment by banesullivan** [comment]
Please rebase as I reverted the `no-ci` checks - sorry for the inconvienance!

**Review by akaszynski — APPROVED** [review]
LGTM. Thanks for adding this!

**Comment by tkoyama010** [comment]
Thanks you for review.

**Pull request merged**